scim.sgnl.yaml

```diff
@@ -0,0 +1,419 @@
+        type: Bool 
+      - name: password
+        externalId: password
+        type: String
+      
+      # Enterprise User Extension
+      - name: employeeNumber
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].employeeNumber
+        type: String
+      - name: costCenter
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].costCenter
+        type: String
+      - name: organization
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].organization
+        type: String
+      - name: division
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].division
+        type: String
+      - name: department
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].department
+        type: String
+      - name: managerId
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.value
+        type: String  
+      - name: managerDisplayName
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.displayName
+        type: String
+      
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version 
+        externalId: $.meta.version
+        type: String  
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  Group:
+    displayName: SCIMGroup
+    externalId: Groups
+    description: Group entity in SCIM
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  GroupMember:
+    parent: Users  # External ID of the parent entity
+    displayName: SCIMGroupMember
+    externalId: groups
+    description: Entity representing SCIM user group members
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String
+      - name: type
+        externalId: type
+        type: String
+      - name: ref
+        externalId: '..["$ref"]'
+        type: String
```

Comment:
The requirement is to create a "MEMBER" relationship between a SCIM User and a SCIM Group so that snippet creators can perform membership checks. Group membership data is ingested as `GroupMember`, a child entity of Entity `User`.

Can you please recommend the correct way to create relationships here.
cc: @rlenglet, @nholbrook